epub33/a11y/index.html

```diff
@@ -1852,74 +1852,88 @@ <h2>Change log</h2>
-				<li>12-Mar-2021: Changed the distribution section to non-normative but added a note that the
-					requirements must be followed where required by law (e.g., in the EU). See <a
-						href="https://github.com/w3c/epub-specs/issues/1487">issue 1487</a>.</li>
-				<li>08-Mar-2021: Add objective for the sequence of <code>par</code> and <code>seq</code> elements in
-					media overlay documents to reflect a logical reading order. See <a
-						href="https://github.com/w3c/epub-specs/issues/1556">issue 1556</a>.</li>
-				<li>05-Mar-2021: Added recommendation that page markers be included for all pages of content reproduced
-					from source and best practice to include markers for all pages in the source. See <a
-						href="https://github.com/w3c/epub-specs/issues/1502">issue 1502</a>.</li>
-				<li>05-Mar-2021: Added recommendation that page list include links to all pages of content reproduced
-					from source and best practice to include links to all pages in the source. See <a
-						href="https://github.com/w3c/epub-specs/issues/1503">issue 1503</a>.</li>
-				<li>05-Mar-2021: Restructured the EPUB Requirements section to split out the individual objectives that
-					were grouped together under the Page Navigation and Media Overlays headings. See <a
-						href="https://github.com/w3c/epub-specs/issues/1458">issue 1458</a>.</li>
-				<li>25-Feb-2021: Replaced the IDPF URLs used to report conformance to the 1.0 specification with more
-					flexible text strings. See <a href="https://github.com/w3c/epub-specs/issues/1455">issue
-					1455</a>.</li>
-				<li>19-Feb-2021: References to WCAG 2.0 have been updated to undated references to WCAG 2 (except where
-					WCAG 2.0 is explicitly mentioned for conformance).</li>
-				<li>01-Feb-2021: Changed the recommendation that media overlays conform to the requirements of the EPUB
-					specification to a requirement. See <a href="https://github.com/w3c/epub-specs/issues/1486">issue
-						1486</a>.</li>
-				<li>17-Nov-2020: Added the <code>refines</code> attribute to the <code>certifierCredential</code> and
-						<code>certifierReport</code> properties and examples. See <a
-						href="https://github.com/w3c/epub-specs/issues/1410">issue 1410</a>.</li>
-				<li>16-Nov-2020: References to the optional accessibility control and API metadata have been removed
-					from the discoverability section. These metadata properties are more applicable to reading systems.
-					See <a href="https://github.com/w3c/epub-specs/issues/1327">issue 1327</a>.</li>
-				<li>26-Sept-2020: The revisions made to the Accessibility 1.0 specification as part of publishing it as
-					an ISO standard have been incorporated into the initial draft text.</li>
-			</ul>
+			<section id="change-latest">
+				<h3>Substantive changes since the <a href="https://www.w3.org/TR/2022/CR-epub-a11y-11-20220512/"
+						>previous Candidate Recommendation Snapshot</a></h3>
+
```

Comment:
Is the idea that if I we have a new snapshot published, then these items are manually moved down to the general list? Or that we'd have a subsection for every CR snapshot? The latter might be an overkill, the former is a bit of a pain at every snapshot publication...

Reply:
The former. We migrate the old changes down to the complete list after the snapshot. Same process we were using before we went to publishing automatic working drafts.

The only headache with the new publishing system is we can't prep the change log immediately after publishing the snapshot because it would be pushed out right away. It has to happen with the first substantive change post-snapshot.

Reply:
Actually, I was just looking at the github documentation and you can temporarily disable the workflows without having to remove them: https://docs.github.com/en/actions/managing-workflow-runs/disabling-and-enabling-a-workflow

We could turn off publication and push the updated change log after the snapshot, which would make life a lot simpler.

Reply:
Well, ya, I think this is useful to get in now regardless so we have a clear record of changes since we went to CR. Even if we don't publish another CR snapshot, it's nice to see what's happened since the transition occurred. If we do publish another snapshot, we can fix the lists then. It's not that complicated.

Reply:
One thing we could do, too, is change the title of the section to "Changes since Candidate Recommendation" (with a link to the first snapshot) and perhaps not worry about every individual snapshot.

Reply:
> Even if we don't publish another CR snapshot, it's nice to see what's happened since the transition occurred.

It may also become handy when we move to PR...